.github/workflows/ci.yaml

```diff
@@ -0,0 +1,77 @@
+name: CI
+on:
+  pull_request:
+  push:
+      branches:
+        - master
+      tags:
+        - "*.*"
+
+env:
```

Comment:
do we need this? 
https://github.blog/changelog/2020-02-21-github-actions-breaking-change-ubuntu-virtual-environments-will-no-longer-start-the-mysql-service-automatically/

Reply:
huh, I would think that we would, but the build passed without it....